go.mod

```diff
@@ -29,3 +29,5 @@ require (
 )
 
 replace k8s.io/client-go => k8s.io/client-go v0.21.2
+
+replace github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.48.1 => github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.49.1-0.20210803083158-78e0ea966b65
```

Comment:
I would stick with v0.49.0 and leave aside Authorization and OAuth2 support for now (it can be added in 4.10 once we upgrade to a version of the Prometheus operator supporting them).

Reply:
Oh yeah, good point ty.